src/components/custom/tooltip-wrapper.jsx

```
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import { Arrow } from '@radix-ui/react-tooltip';
+import { useState } from 'react';
+import { TooltipProvider, Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip';
+import { Button } from '@/components/ui/button';
+
+export function TooltipWrapper({ children, content, side = 'bottom', className, ...props }) {
```

ESLint

[error] 'className' is defined but never used.

(no-unused-vars)